platform/linux-generic/odp_ishm.c

```diff
@@ -238,13 +238,150 @@ typedef struct {
 } ishm_ftable_t;
 static ishm_ftable_t *ishm_ftbl;
 
+#define HP_CACHE_SIZE 32
+struct huge_page_cache {
+	uint64_t len;
+	int total; /* index in fd that's the highest allocated */
+	int idx;   /* retrieve fd[idx] to get a free file descriptor */
+	unsigned int seq_num;
```

Comment:
It seems there is no apparent reason to store `seq_num` here.

Reply:
agreed, it is not necessary